### ISSUE TITLE
the installation test of pipeline-umi-amplicon failed

### ISSUE THREAD

**Issue by dieunelderilus** [nanoporetech/pipeline-umi-amplicon#11]
Hello-  I have followed properly the instructions provided to install  this pipeline-umi-amplicon. However, the installation test  failed Command used:
`snakemake -j 1 -pr --configfile config.yml`

Screenshot of the error message:

![image](https://user-images.githubusercontent.com/26675022/127336595-52a57db7-1436-434e-be77-f43fdb98e75c.png)

Please any help?
~DD

**Comment by philres** [comment]
Hi and thanks for reporting this.

Could you pleas try activating the conda environment and then running:
```
pip install numpy==1.19.5
```

and then try running:
```
snakemake -j 1 -pr --configfile config.yml
```
again?

Thanks,
Philipp

**Comment by dieunelderilus** [comment]
Hello Philip,

Here is  is the output of the pip intall command:
`(pipeline-umi-amplicon) [qlk5@biolinux pipeline-umi-amplicon]$ pip install numpy==1.19.5
Requirement already satisfied: numpy==1.19.5 in /scicomp/home- pure/qlk5/Desktop/important_programs/miniconda3/envs/pipeline-umi-amplicon/lib/python3.8/site-packages (1.19.5)`

Here is the output of  of the  `snakemake -j 1 -pr --configfile config.yml`  command:
![image](https://user-images.githubusercontent.com/26675022/129428971-8fb068de-cad3-4ce4-846c-ad5155622e38.png)

I have  tried this in 2 different computers  (cluster) but I still have the same  error message at the end.

Any suggestions?
Thanks in advance!
~DD

**Comment by philres** [comment]
HI again and sorry for not getting back to you earlier.

Could you please post the content of this file:
<img width="811" alt="Screenshot 2021-09-16 at 18 32 59" src="https://user-images.githubusercontent.com/2323996/133650741-b96847cd-7ac0-4ede-929e-515d4ad0cbcf.png">
here?

Thanks,
Philipp

**Comment by KeelyDulmage** [comment]
Hello, just wanted to say that I had that same failed test error. I ran `pip install numpy==1.19.5` before rerunning the test and it successfully fixed the issue on my end.

**Comment by dieunelderilus** [comment]
Hello @KeelyDulmage - Unfortunately I  have tried your suggestion but it  doesn't work in my computer. I have numpy==1.19.5  installed  and running in my working environment.
Thank you
~DD

**Comment by scogi** [comment]
Same for me, installing numpy 1.19.5 did not resolve the problem (see also #13  )

**Comment by dieunelderilus** [comment]
> Same for me, installing numpy 1.19.5 did not resolve the problem (see also #13 )
Yes this is problematic, please let me know if you fix the  issue!

**Comment by dieunelderilus** [comment]
> Same for me, installing numpy 1.19.5 did not resolve the problem (see also #13 )
This is strange , they have closed this issue  while it is still a serious problem for the user. I still don't know how to solve this.
Cheers,
~DD

**Comment by passelma42** [comment]
> > Same for me, installing numpy 1.19.5 did not resolve the problem (see also #13 )
> > This is strange , they have closed this issue  while it is still a serious problem for the user. I still don't know how to solve this.
> > Cheers,
> > ~DD

Just opened a new issue, pip install numpy==1.19.5 generates a numerous of dependency issues. Original versions in the environmen.yml file were not given. Building the conda env based on this file, will give you the latest versions of the predefined tools. Some of these tools need numpy 1.24.4 ... .
Running the pipeline as it is, takes you as far as generating unpolished UMI consensuses and some stats... . Better than nothing, but I hope they will resolve the issues. (Or look for another tool which can handle UMI amplicons)
